apps/server/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "server",
+  "version": "0.0.8-beta",
+  "packageManager": "npm@10.5.0",
+  "scripts": {
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run",
+    "dev": "ts-node src/server.js",
+    "dev:debug": "node --inspect -r ts-node/register ./src/server.js",
+    "server": "npm run dev",
+    "lint": "eslint . --ext .js,.ts",
+    "lint:fix": "eslint . --fix --ext .js,.ts",
+    "coverage": "npm test -- --verbose --coverage --watchAll",
+    "test": "jest --no-coverage",
+    "test:watch": "jest --no-coverage --watch",
+    "test-clear": "npm test -- --clearCache",
+    "keys": "node -e 'console.log(`key: ${crypto.randomBytes(32).toString(\"hex\")}\n IV: ${crypto.randomBytes(16).toString(\"hex\")}`)'",
+    "precommit": "lint-staged",
+    "prepare": "husky"
+  },
+  "dependencies": {
+    "@babel/eslint-parser": "^7.18.2",
+    "@capacitor-community/http": "^1.4.1",
+    "@types/express": "^4.17.21",
+    "assert-browserify": "^2.0.0",
+    "axios": "^1.6.8",
+    "buffer-browserify": "^0.2.5",
+    "crypto-browserify": "^3.12.0",
+    "crypto-js": "^4.2.0",
+    "dotenv": "^16.3.1",
+    "express": "^4.19.2",
+    "express-async-errors": "^3.1.1",
+    "express-rate-limit": "^7.0.2",
+    "husky": "^9.0.11",
+    "js-logger": "^1.6.1",
+    "redis": "^4.6.12",
+    "stream-browserify": "^3.0.0",
+    "uuid": "^9.0.1"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "@babel/core": "^7.21.3",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
+    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
+    "@babel/preset-env": "^7.23.8",
+    "@babel/preset-react": "^7.18.6",
+    "@testing-library/cypress": "^10.0.1",
+    "@types/jest": "^29.5.12",
+    "@typescript-eslint/eslint-plugin": "^6.20.0",
+    "babel-plugin-js-logger": "^1.0.17",
+    "customize-cra": "^1.0.0",
+    "cypress": "^13.7.1",
+    "eslint": "^8.56.0",
+    "eslint-config-mx": "^4.0.0",
+    "eslint-config-standard-with-typescript": "^43.0.1",
+    "eslint-plugin-cypress": "^2.15.1",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jest": "^27.6.3",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-promise": "^6.1.1",
+    "lint-staged": "^15.2.2",
+    "msw": "^2.2.13",
+    "prettier": "^3.2.5",
+    "ts-jest": "^29.1.2",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.3.3"
+  },
+  "lint-staged": {
+    "{.eslintrc.js,.prettierrc.js}": [
+      "prettier --write"
+    ],
+    "*.{js[x],ts[x]}": [
+      "npm run lint -- --cache --fix"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "prettier --write . && lint-staged && npm run test && git add -A ."
```

Comment:
Are these hooks working? I would guess that they need to be in the root package.json to work

Reply:
Good catch. I'll take a look.